### DISCUSSION TITLE
pyMOR Community Meeting 2024-02-07

### DISCUSSION THREAD

**Discussion by sdrave** [pymor/pymor#2254] [Announcements]
The next pyMOR Community Meeting will take place Wednesday, February 7, at 15:00 CET and 9:00 EST, online at:

Link: http://bbb.mpi-magdeburg.mpg.de/meeting/pymor
Password: THaEo

Anyone interested in pyMOR can join.

Currently on the agenda are:

- [ ] ...
- [ ] Community Meeting frequency (@sdrave)
- [ ] Strategy meeting
- [ ] AOB (@sdrave)
- [ ] Set new release date (@lbalicki)

Moderator: @sdrave

**Comment by lbalicki** [comment]
Notes:

- Community Meeting frequency
  - meet less regularly unless we get closer to release
  - decided on every 3 weeks for now
- Strategy meeting
  - many points with main discussion about what to do with vector arrays
  - embracing vector arrays more could enable quasi matrices, chebfun, low-rank tensor things, ...
  - next strategy meeting: In 2 weeks 14:00 CET, 8:00 EST
- Mention resigned main developers in the release notes
- New Release Date for 2024.1. : July 5th

Next Community meeting: 14:00 CET, 8:00 EST, 02/29/2024
